app/controllers/catalog_controller.rb

```diff
@@ -1776,7 +1777,7 @@ def get_node_info_handle_stc_node(id)
 
   def get_node_info_handle_leaf_node_stcat(id)
     @record = ServiceTemplateCatalog.find(id)
-    @record_service_templates = Rbac.filtered(@record.service_templates)
+    @record_service_templates = Rbac.filtered(@record.service_templates.where("service_type != 'internal'"))
```

Comment:
@h-kataria I think you want to use the new `scope` here instead of recreating the `where` clause.

Reply:
@gmcculloug will clean those up.